airflow/providers/google/cloud/example_dags/example_dataproc.py

```diff
@@ -65,10 +65,16 @@
 # Update options
 # [START how_to_cloud_dataproc_updatemask_cluster_operator]
 CLUSTER_UPDATE = {
-    "config": {"worker_config": {"num_instances": 3}, "secondary_worker_config": {"num_instances": 3}}
```

Comment:
How is adding kafka integration related to modifying google dag examples?

Reply:
idk how those even got in.  Whenever i can work on this, i rebase from master and a whole bunch of changes get added..  i think I'm rebasing wrong but i don't understand how.  `git rebase -i <my_branch> master`?

I never have enough time to really isolate the issue with this PR, especially since everytime i try toupdate my local branch i incur other commits that I didn't write.  not sure what's wrong with this at all.

Reply:
There is a really good guide at the contributing section:
https://github.com/apache/airflow/blob/master/CONTRIBUTING.rst#id9
On slack there is a channel airflow-how-to-pr where you can ask for help if you get into trouble.

Reply:
i've read through this guide and i'm still not sure what's happening here.  I've joined the slack channel and raised my concerns there as well